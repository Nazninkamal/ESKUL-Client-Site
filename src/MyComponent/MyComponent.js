import React from 'react';

const MyComponent = () => {
    return (
        <div>
            <h1>heloo world</h1>
        </div>
    );
};

export default MyComponent;