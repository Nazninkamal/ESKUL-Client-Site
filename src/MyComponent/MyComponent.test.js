import React from 'react';
import { shallow } from 'enzyme';
import MyComponent from './MyComponent';


// component test

describe('MyComponent test', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<MyComponent/>);
    expect(component).toMatchSnapshot();
  });

  //
});

