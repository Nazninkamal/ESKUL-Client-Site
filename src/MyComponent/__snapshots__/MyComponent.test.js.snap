// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`MyComponent test should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
